elated-egrets/bangazon-internal-site-2.0
# Rm local product

## DESCRIPTION
added ability for sellers to tag items as available for local delivery, this then pulls from the city field on the sellers profile

## How to pull down
Commands

```
git fetch --all
git checkout rm-local-product
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. remake migrations
1. create a new user and make sure you add a city to the profile
1. make a couple of new items, mark one for local delivery and the other as not
1. go to each products detail page, the one specified for local delivery should say its available in the city of your user profile, and the other should not specify anything to do with location. 

## What you changed
added field to product model for making products available for local delivery. and made the product detail template list the city of the seller if the product is marked this way. 

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
